tests/test_funcs.py

```diff
@@ -509,40 +509,21 @@ def test__xy_south_of_limit(lat):
     assert y > 1
 
 
-def test_x_minmax():
-    assert mercantile.x_minmax(zoom=0) == (0, 0)
-    assert mercantile.x_minmax(zoom=1) == (0, 1)
+def test_minmax():
+    """Exercise minmax from zoom levels 0 to 28"""
+    assert mercantile.minmax(zoom=0) == (0, 0)
+    assert mercantile.minmax(zoom=1) == (0, 1)
 
     for z in range(0, 28):
-        xmin, xmax = mercantile.x_minmax(z)
+        minimum, maximum = mercantile.minmax(z)
 
-        assert xmin == 0
-        assert xmax >= 0
-        assert xmax == 2 ** z - 1
+        assert minimum == 0
+        assert maximum >= 0
+        assert maximum == 2 ** z - 1
 
 
-def test_y_minmax():
-    assert mercantile.y_minmax(zoom=0) == (0, 0)
-    assert mercantile.y_minmax(zoom=1) == (0, 1)
-
-    for z in range(0, 28):
-        ymin, ymax = mercantile.y_minmax(z)
-
-        assert ymin == 0
-        assert ymax >= 0
-        assert ymax == 2 ** z - 1
-
-
-def test_xy_minmax():
-    for z in range(0, 28):
-        xmin, xmax = mercantile.x_minmax(z)
-        ymin, ymax = mercantile.y_minmax(z)
-
-        assert xmin == 0
-        assert ymin == 0
-
-        nx = xmin + xmax + 1
-        ny = ymin + ymax + 1
-
-        assert nx == 2 ** z
-        assert ny == 2 ** z
+@pytest.mark.parametrize("z", [1.2, "lol", -1])
```

Comment:
Test of the input validation.

Reply:
I'm not deep into Python testing, but do we need to test and check for wrong type?

And if we do, should it then be an assertion in the function, to assert that we get an int (and/or float) from the user? It's not an exception, but rather a pre-condition the user violated, no?

Reply:
Yes. Maybe it's better if I don't conflate invalid value and invalid type.